### PR TITLE
NCDA activity - Group encounter

### PR DESCRIPTION
**Pull request by anvmn**
#464

### CODE REVIEW

**Comment by anvmn** [comment]
Peek 2022-10-12 15-47

**Comment by anvmn** [comment]
@AronNovak please review

File: client/src/elm/Measurement/Utils.elm [hunk 1]
```diff
@@ -3858,3 +3882,241 @@ toNCDAValue form =
...
+
+        -- In how many days (from the session date) will the child be 2 years
+        -- old?
+        daysUntilTwoYearsOld =
+            (365 * 2) - age
+
+        -- In how many days (from the session date) will the child be 1 year
+        -- old?
+        daysUntilOneYearOld =
+            365 - age
```
Comment (AronNovak) [inline client/src/elm/Measurement/Utils.elm:3984]: > In the Julian calendar, a year contains either 365 or 366 days, and the average is 365.25 calendar days.

It's this or 366, but I guess it's ok here.

**Review by AronNovak — APPROVED** [review]

File: client/src/elm/Measurement/Utils.elm (hunk 1 above)
Comment (anvmn) [reply client/src/elm/Measurement/Utils.elm:3984]: Well, this is very old code which I had to move.
I don't think it makes much difference, but the code is not in use anyway.

**Pull request merged**